wordpress-mobile/WordPress-iOS
# Prevent rotation of Blogging Reminders flow

This PR pull in the latest version of WordPress-UI, which allows the blogging reminders flow to stop itself being rotated.

**To test**

* Enable the blogging reminders feature flag, and add a trigger somewhere to show the flow
* Build and run and view the flow on iPhone
* Try rotating your device / simulator and ensure the flow doesn't rotate
* Go to the editor and bring up the prepublishing sheet. Ensure this _does_ still rotate

## Regression Notes

1. Potential unintended areas of impact

Other bottom sheet views

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Ensured they still rotate, as above

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:
